Samples~/SampleGame/README.md

```diff
@@ -1,147 +1,69 @@
+* Running the client or server code within the Unity Editor.
 
-## Setting up the game client
+- [Prerequisites](#prerequisites)
+- [Integration with the deployment scenarios](#integration-with-the-deployment-scenarios)
+- [Integration with Amazon GameLift Anywhere](#integration-with-amazon-gamelift-anywhere)
+- [Playing the game](#playing-the-game)
+- [Running tests](#running-tests)
 
-The client will need the AWS Region, Cognito Client ID and API Gateway Endpoint. They can be set in Unity, the
-scriptable object is at `Assets\Settings\GameLiftClientSettings`. When the scenario is deployed, its outputs are also
-displayed in the Deployment window. Copy them out of the window and paste in the GameLiftClientSettings. Anyone playing
-against you on a different machine should use the same settings to join your server, so you could distribute the client,
-if you wanted that.
+## Prerequisites
 
-Make sure that the scenes are in the "Scenes In Build" list in the File/Build Settings window. For running the smaple in
-Unity editor, start the BootstrapScene scene.
+Follow the plugin [prerequisites](../../README.md#prerequisites) and [install instructions](../../README.md#install-the-plugin).
+* Make sure to install a compatible Server SDK version (C# Server SDK for Unity plugin v5.x).
+* Make sure to initialize the plugin settings after successfully importing the sample project.
+    1. In your Unity project, select `Amazon GameLift > Sample Game > Import Sample Game` and import all assets.
+    1. In your Unity project, select `Amazon GameLift > Sample Game > Initialize Settings`.
 
-## Building the project
+## Building the Client vs the Server
 
-1. To build client, just open the File/Build Settings window, make sure you have "Server Build" unchecked, and press "
-   Build". Select a client build folder.
-1. To build server, in the File/Build Settings window, press "Player Settings", and add a UNITY_SERVER to Scripting
-   Define Symbols. Restart Unity. Open the File/Build Settings window, make sure you have "Server Build" checked, and
-   press "Build". Select a server build folder.
+The sample project uses the `UNITY_SERVER` preprocessor to distinguish between client and server build targets. When you do `Initialize Settings`, this preprocessor is set up for the dedicated server build target. As long as the sample has been initialized, switching between build targets is enough to ensure the correct code path is compiled.
 
-You can also use "GameLift / Configure Sample Server" and "GameLift / Configure Sample Client" in Unity main menu.
+## Integration with the deployment scenarios
 
-## How to deploy a server to GameLift
+The client will need the AWS Region, Cognito Client ID and API Gateway Endpoint which are stored in the scriptable object at `Assets\Settings\GameLiftClientSettings.asset`. When the scenario is deployed, these properties are imported automatically when clicking `Apply Managed EC2 Settings` from the `Host with Managed EC2` tab.
```

Comment:
I don't see an equivalent line for `Apply Anywhere settings`, am i misreading that?

Reply:
Gotcha, adding the following: `After creating an Anywhere fleet and registering compute, this property is imported automatically when clicking `Apply Anywhere Settings` from the `Host with Anywhere` tab.`

Reply:
sounds good